app/inpututils.cpp

```diff
@@ -544,7 +546,19 @@ bool InputUtils::acquireCameraPermission()
   {
     return mAndroidUtils->requestCameraPermission();
   }
-  return true;
+  else
+  {
+    auto status = qApp->checkPermission( QCameraPermission{} );
+
+    if ( Qt::PermissionStatus::Undetermined == status )
```

Comment:
This will be handled by @tomasMizera in follow up PR properly (along with location permissions)